src/riak_pb_ts_codec.erl

```diff
@@ -170,6 +172,5 @@ decode_cells([#tscell{binary_value    = undefined,
                       timestamp_value = undefined,
                       boolean_value   = undefined,
                       double_value    = undefined} | T], Acc) ->
-    %% all fields undefined: what are the circumstances where it is
-    %% possible/permitted?
+    %% NULL Cell
     decode_cells(T, [[] | Acc]).
```

Comment:
It's inline with the other recursive calls (see line [170](https://github.com/basho/riak_pb/pull/152#discussion-diff-43544377R170))